@xen-orchestra/web/src/pages/vm/new.vue

```diff
@@ -6,6 +6,12 @@
         <VtsSelect :id="poolSelectId" accent="brand" />
       </template>
     </UiHeadBar>
+    <UiAlert v-if="vmState.pool" accent="info">
+      {{ t('new-vm.feature-not-supported') }}
+      <UiLink :href="xo5Link" size="medium">
+        {{ t('xo-5') }}
+      </UiLink>
+    </UiAlert>
```

Comment:
Here is the usage of the VueI18n component:
```suggestion
    <UiAlert v-if="vmState.pool" accent="info">
      <I18nT keypath="new-vm.feature-not-supported">
        <template #xo-5>
          <UiLink :href="xo5Link" size="medium">
            {{ t('xo-5') }}
          </UiLink>
        </template>
      </I18nT>
    </UiAlert>
```
The `{xo-5}` string defined in the translations is the _slot name_